examples/mixture_models/dependent_density_regression.ipynb

```diff
@@ -2,16 +2,7 @@
  "cells": [
```

Comment:
<p>You can use <code>"one": [1]</code>  in <code>coords</code>  dict, or any other name/coord value you may prefer, it just has to be there and have length one to act as the <code>1</code>  in shape.</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc-examples/pull/146/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

Reply:
<p>so simple, thanks!</p>